.github/workflows/continuous-deploy.yml

```diff
@@ -37,6 +37,10 @@ jobs:
       packages: write # To push to GHCR.io
     runs-on: ubuntu-latest
 
+    concurrency:
+      group: cicd-build-${{ matrix.image }}}
+      cancel-in-progress: false
```

Comment:
should this also be true?
or is this covered by the first setting:
```
  group: cicd-${{ github.ref }}
  cancel-in-progress: true
```

Reply:
No, this overrides the global concurrency config.  We don't want the same images to be built concurrently, it messes with the cache.

Reply:
Aah, right. I'm used to the build job including the pytest job :+1:

Reply:
We could configure concurrency per branch, where `cicd-build-${{ github.ref }}-${{ matrix.image }}`, and cancel in progress, but I want to test some stuff first, because the caching is fickle.